RInAppMessaging/Classes/Utilities/AtomicWrapper.swift

```diff
@@ -1,7 +1,8 @@
 /// This wrapper ensures synchronized access to the value only for getter and setter.
-/// Mutating functions, subscript, incrementation etc. are not synchronized.
+/// Mutating functions, subscript, incrementation etc. are not synchronized by default -
+/// use `mutate` function to ensure operation atomicity.
 @propertyWrapper
-struct AtomicGetSet<Value> {
+class AtomicGetSet<Value> {
```

Comment:
Not possible to do the mutate as a struct?

Reply:
Hmm I got some errors but it's seems that it's working with `struct` after all. I was also wondering if shadow copying the property because of `inout` in mutate() is going to cause any problems but I think it's ok because everything is synced